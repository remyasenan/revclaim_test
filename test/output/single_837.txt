ISA*00*          *00*          *ZZ*412345678      *ZZ*ANS            *080222*0106*U*00401*000002833*0*P*<~
GS*HC*412006324*ANS*20080222*0106*2831*X*004010X098A1~
ST*837*0001~
BHT*0019*00**20080401*010018*CH~
REF*87*004010X098A1~
NM1*41*2*REVENUEMED*****46*412345678~
PER*IC*JOE WU*EM*JWU@REV.COM~
NM1*40*2*ANS*****46*ANS~
HL*1**20*1~
NM1*85*2*RADIOLOGY AND IMAGING ASSOC*****XX*1013901222~
N3*LLC PO BOX~
N4*MEMPHIS*TN*38174~
REF*EI*621498232~
REF*G2*11498232~
HL*2*1*22*0~
SBR*P*18*NONE*UNITED AMERICAN HEALTH CORP*****CI~
NM1*IL*1*ECWEKS*BELENDA****MI*414138335~
N3*995 KORTH WATKINS ART 4~
N4*MEMPHIS*TN*38126~
DMG*D8*19610617*F~
NM1*PR*2*ANS*****PI*ANS~
CLM*5003269447*111***11<<1*Y*A*Y*Y*B~
REF*D9*~
REF*EA*5003269447~
HI*BK<715.80*BF401.1~
NM1*DN*2*,*****XX*~
PRV*RF*ZZ*208D00000X~
NM1*82*1*OLAWALE*MORAFA*MD***XX*1013901222~
PRV*PE*ZZ*208D00000X~
REF*EI*621498232~
REF*G2*612498232~
NM1*77*2*HEADTHFIRST FAMILY CARE*****XX*~
N3*1325 BASTNORELAND AVE~
N4*MEMPHIS*TN*38104~
REF*EI*621498232~
LX*1~
SV1*HC<99214*111*UN*1***1<2~
DTP*472*RD8*20080130-20080130~
NM1*82*2*NPI 1013901222*****XX*1013901222~
PRV*PE*ZZ*208D00000X~
REF*G2*612498232~
SE*40*0001~
GE*1*2831~
IEA*1*000002833~
