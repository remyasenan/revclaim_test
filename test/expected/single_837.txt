ISA*00*          *00*          *ZZ*412345678      *ZZ*ANS            *080501*1922*U*00401*000000001*0*P*:~
GS*HC*412006324*ANS*20080501*1922*2831*X*004010X098A1~
ST*837*0001~
BHT*0019*00*20080403678*20080501*1922*CH~
REF*87*004010X098A1~
NM1*41*2*REVENUEMED*****46*412345678~
PER*IC*Joe Wu*EM*ans@revenuemed.com~
NM1*40*2*ANS*****46*ANS~
HL*1**20*1~
NM1*85*2*RADIOLOGY AND IMAGING ASSOC*****XX*1013901222~
N3*LLC PO BOX~
N4*MEMPHIS*TN*38174~
REF*EI*621498232~
REF*G2*2611498232~
HL*2*1*22*0~
SBR*P*18*9*UNITED AMERICAN HEALTH CORP*****CI~
NM1*IL*1*ECWEKS*BELENDA****MI*414138335~
N3*995 KORTH WATKINS ART 4~
N4*MEMPHIS*TN*38126~
DMG*D8*19610617*F~
NM1*PR*2*ANS*****PI*ANS~
CLM*5003269447*111***11::1*Y*A*Y*Y*B~
REF*D9*20080403678~
REF*EA*5003269447~
HI*BK:715.80*BF:401.1~
NM1*82*1*OLAWALE*MORAFA*MD**A*XX*1013901222~
PRV*PE*ZZ*208D00000X~
REF*EI*621498232~
REF*G2*612498232~
NM1*77*2*HEADTHFIRST FAMILY CARE~
N3*1325 BASTNORELAND AVE~
N4*MEMPHIS*TN*38104~
REF*EI*621498232~
LX*1~
SV1*HC:99214*111*UN*1*11**1:2~
DTP*472*RD8*20080130-20080130~
NM1*82*2*NPI 1013901222*****XX*1013901222~
PRV*PE*ZZ*208D00000X~
REF*G2*612498232~
SE*38*0001~
GE*1*2831~
IEA*1*000000001~
